roles/swift-account/handlers/main.yml

```diff
@@ -4,8 +4,8 @@
     name: "{{ item }}"
     state: restarted
   with_items:
-    - swift-account
-    - swift-account-auditor
-    - swift-account-reaper
-    - swift-account-replicator
+    - "{{ (ansible_os_family == 'RedHat') | ternary('openstack-', '') }}swift-account"
```

Comment:
Note: I stopped commenting mid way as later half has same comments. Please address them and will review it. At high level, please follow flow we are using in other roles, glance, nova, neutron, heat, aodh, ceilometer, cinder, etc. 

1: services to be defined in openstack-meta, swift role defaults defined service variable (please follow same variable hierarchy as other openstack roles), and handlers, service creation, start, stop etc uses the role defaults variable. No need to duplicate the tasks for os == 'rhel' or os == 'ubuntu' single task will handle it.

2: Most places code can be optimized but taking below 2 things into considerations
    1: servicenames, paths, etc which are only difference in requiring to create duplicate tasks per os family (rhel or ubuntu) should be abstracted out to role defaults and leverage single task
    2: try to block tasks where possible for os == 'ubnutu' and os == 'rhel' . This will help in futrue for others to follow the code and maintain it.

3: be consistent with other roles.. we use os == ' ubuntu'  (for os family debian) and os == 'rhel' (for os family Rhel).